packages/suite/src/views/wallet/send/components/Options/BitcoinOptions/CoinControl/UtxoSearch.tsx

```diff
@@ -0,0 +1,81 @@
+import { useCallback, useRef, Dispatch, SetStateAction, ChangeEvent, KeyboardEvent } from 'react';
+import styled, { useTheme } from 'styled-components';
+import { Input, Icon, KEYBOARD_CODE } from '@trezor/components';
+import { useTranslation } from 'src/hooks/suite/useTranslation';
+import { isFeatureFlagEnabled } from '@suite-common/suite-utils';
+import { borders, spacingsPx } from '@trezor/theme';
+
+const Container = styled.div`
+    display: flex;
+    align-items: center;
+    width: 100%;
+`;
+
+const StyledInput = styled(Input)`
+    width: 100%;
+    margin-right: ${spacingsPx.xs};
+    border-radius: ${borders.radii.full};
+
+    input {
+        height: 38px;
+        width: 100%;
+        border: none;
+    }
+`;
+
+const SearchIcon = styled(Icon)`
+    cursor: pointer;
+`;
+
+export type UtxoSearchProps = {
+    searchQuery: string;
+    setSearch: Dispatch<SetStateAction<string>>;
+    setSelectedPage: Dispatch<SetStateAction<number>>;
+};
+
+export const UtxoSearch = ({ searchQuery, setSearch, setSelectedPage }: UtxoSearchProps) => {
+    const theme = useTheme();
+    const inputRef = useRef<HTMLInputElement | null>(null);
+    const { translationString } = useTranslation();
+
+    const onKeyDown = useCallback(
+        (event: KeyboardEvent) => {
+            // Handle ESC (un-focus)
+            if (event.code === KEYBOARD_CODE.ESCAPE && inputRef.current) {
+                setSearch('');
+                inputRef.current.blur();
+            }
+        },
+        [setSearch],
+    );
+
+    const onSearch = useCallback(
+        ({ target }: ChangeEvent<HTMLInputElement>) => {
+            setSearch(target.value);
+            setSelectedPage(1);
+        },
+        [setSearch, setSelectedPage],
+    );
+
+    if (!isFeatureFlagEnabled('SEARCH_TRANSACTIONS')) {
```

Comment:
I think we can remove this. This is not about transactions. 



Btw @MiroslavProchazka do you know why we have this for transactions? Its some kind of a a code-feature where we can turn Tx-search off, but I think we can (and should) remove it completely. The `SEARCH_TRANSACTIONS` feature flag seems to me as the relict of the past.